releases/release-1.22/release-notes/maps/pr-101496-map.yaml

```diff
@@ -1,4 +1,4 @@
 pr: 101496
 releasenote:
-  text: `podAffinity`, `NamespaceSelector` and the associated `CrossNamespaceAffinity`
-    quota scope features graduate to Beta and they are now enabled by default.
+  text: "`podAffinity`, `NamespaceSelector` and the associated `CrossNamespaceAffinity`
```

Comment:
Can you put a word before  `podAffinity`, something like The or A ?

Reply:
Updated. Would you mind to review again? Thank you